candi/bashible/common-steps/node-group/064_configure_kubelet.sh.tpl

```diff
@@ -308,9 +308,15 @@ systemReserved:
   ephemeral-storage: 1Gi
 {{- else if eq $resourceReservationMode "Static" }}
 systemReserved:
-  cpu: {{ dig "kubelet" "resourceReservation" "static" "cpu" 0 .nodeGroup }}
-  memory: {{ dig "kubelet" "resourceReservation" "static" "memory" 0 .nodeGroup }}
-  ephemeral-storage: {{ dig "kubelet" "resourceReservation" "static" "ephemeralStorage" 0 .nodeGroup }}
+  {{- if .nodeGroup.kubelet.resourceReservation.static.cpu }}
```

Comment:
This check has hidden logic. We need to check all keys in the chain - 
kubelet, resourceReservation, static

Reply:
Replaced with a chain from hasKey